massdriver.yaml

```diff
@@ -176,6 +177,11 @@ params:
               const: AOF_DUAL
             - title: RDB
               const: RDB
+        non_ssl_port:
+          type: boolean
+          title: Enable Non-SSL port
+          description: Enable the non-SSL (6379) port. SSL port (6380) is enabled by default. **Enabling this is not recommended as it is less secure**.
```

Comment:
```suggestion
          description: Enable the non-SSL (6379) port. SSL port (6380) is enabled by default. **Enabling this is not recommended as it will allow unencrypted communication, which poses a security risk**.
```